src/test/java/com/cdancy/bitbucket/rest/features/BranchApiLiveTest.java

```diff
@@ -136,12 +140,49 @@ public void testDeleteBranchPermission() {
         }
     }
 
+    @Test(dependsOnMethods = {"testCreateBranch", "testListBranches"})
+    public void testGetBranchModelConfiguration() {
+        branchModelConfiguration = api().getModelConfiguration(projectKey, repoKey);
+        checkDefaultBranchConfiguration();
```

Comment:
If we don't need to do this check twice than lets remove the `checkDefaultBranchConfiguration` method and instead add that code here.